conf/production.ini

```diff
@@ -15,6 +15,8 @@ use: egg:h
 #h.client_id:
 #h.client_secret:
 
+h.hashids.salt: production salt
```

Comment:
Iirc correctly I put this in because Travis was crashing otherwise (it builds the Chrome extension with production.ini iirc). We could fix the Travis script instead